portality/static/js/application_form.js

```diff
@@ -440,7 +440,13 @@ doaj.af.newEditorialApplicationForm = function(params) {
 doaj.af.EditorialApplicationForm = class extends doaj.af.BaseApplicationForm {
     constructor(params) {
         super(params);
-
+        this.form.posted = edges.getParam(params.posted, false)
```

Comment:
Something feels wrong about this.  It looks like the back-end is required to do something in order for the front end to figure out when the form is submitted.  Can't we just switch the state on `submitapplication`?

Reply:
I cannot see how. The idea is that we do not want the application to scroll when the form was re-rendered. When the form is submitted it is sent to the backend and then rerendered from there but the frontend doesn't differentiate where the render method was sent from (get or post part of the method). Even if we would add the hidden field to the form with some flag at the submit it would have to be cleared by the backend when the form is rendered for the first time at the next session. Unless I'm missing something?